src/rules/comment-no-empty/__tests__/index.js

```diff
@@ -0,0 +1,104 @@
+import {
+  messages,
+  ruleName,
+} from ".."
+import rules from "../../../rules"
+import { testRule } from "../../../testUtils"
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+  skipBasicChecks: true,
+
+  accept: [ {
```

Comment:
The focus of these tests should be comments and what's considered empty. It's not clear what's being tested as there's two much non-comment code